lib/srv/db/mongodb/protocol/opcompressed.go

```diff
@@ -114,6 +114,10 @@ func readOpCompressed(header MessageHeader, payload []byte) (message *MessageOpC
 		CompressedMessage: compressedMessage,
 		bytes:             append(header.bytes[:], payload...),
 	}
+	if uncompressedSize <= 0 || len(compressedMessage) <= 0 {
```

Comment:
```suggestion
	if uncompressedSize <= 0 || len(compressedMessage) == 0 {
```

`len` will never return a negative number, right?